src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -200,19 +202,18 @@ def check_fs(self, fs_name):
         fs_map = self.mgr.get('fs_map')
         return fs_name in [fs['mdsmap']['fs_name'] for fs in fs_map['filesystems']]
 
-    def check_pseudo_path(self, pseudo_path):
+    def _fetch_export(self, pseudo_path):
```

Comment:
here exports are fetched from in-memory. what happens when module restarts? shouldn't you be fetching the exports from persistent storage, i.e., the RADOS export objects.

Reply:
You are right. I will look into fixing it.